packages/create-react-app/createReactApp.js

```diff
@@ -272,6 +294,23 @@ function createApp(
     }
   }
 
+  if (useTypeScript) {
+    console.log(
+      chalk.yellow(
+        'The --typescript option has been deprecated and will be removed in a future release.'
```

Comment:
Should this message link to the templates documentation? Same with https://github.com/facebook/create-react-app/pull/7716/files#diff-f9867c1e09ced1328f2ccdac4afac4a5R88